docs/howto/getting-started/change-cust-billing.md

```diff
@@ -0,0 +1,31 @@
+# How to change company information in your account
+
+Via {{gui}} <https://{{gui_domain}}> you can change contact person, adress, company name, purchase order or add c/o adress. This guide will show you how.
```

Comment:
```markdown
Via the [{{gui}}](https://{{gui_domain}}), you ...
```

Also, typo: missing "d" in "address" (twice).

Also, remove the sentence "This guide will show you how"; it is obvious and redundant.